muxers/mplex/src/config.rs

```diff
@@ -84,6 +88,17 @@ impl MplexConfig {
         self.split_send_size = size;
         self
     }
+
+    /// Gets protocol
+    pub fn protocol(&self) -> &[u8] {
+        self.protocol
+    }
```

Comment:
None of the other fields have a _getter_ method. Why is this one needed?

Reply:
I believe it's useful for debugging/logging purposes to be able to retrieve the protocol name when it can be customized